src/du/du.rs

```diff
@@ -289,18 +278,26 @@ pub fn uumain(args: Vec<String>) -> i32 {
     };
 
     let convert_size = |size: u64| -> String {
+        let multiplier: u64 = if matches.opt_present("si") {
+            1000
+        } else {
+            1024
+        };
+        // TODO: Suport Z & Y (currently limited by size of u64)
+        let units = vec![("E", 6), ("P", 5), ("T", 4), ("G", 3), ("M", 2), ("K", 1)];
```

Comment:
Another quick thing.  This could just be a static array instead of a dynamic one.